package.json

```diff
@@ -36,6 +34,9 @@
     "should": "^11.1.2",
     "standard-version": "^4.0.0"
   },
+  "peerDependencies": {
+    "webpack": "^4.0.0"
```

Comment:
Not sure about this. The codebase does not `require('webpack')`. I understand that you added this to avoid runtime errors when webpack's version is not the expected one but I don't think this is the correct solution.

Reply:
@ooflorent we use this for all webpack-contrib repos, better create issue in `webpack-default` about this.

Reply:
@ooflorent Also it is prevent problem with using webpack api between version, if somebody want use `css-loader` on `webpack@3`, they can use old version

Reply:
yeah it's a standard for webpack-defaults. that peerDep on webpack has been in defaults for a very long time. along with the `Requirements` section in the defaults README, this serves as a warning to keep people on supported versions of webpack. new major versions of loaders/plugins/utils are webpack@4+ and Node 6+ only.